library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -447,6 +433,19 @@ private void init() {
         mEventHeaderTextPaint.setColor(getResources().getColor(android.R.color.black));
     }
 
+    public void initEmptyView() {
+//        Rect emptyViewText = new Rect();
+//        String emptyViewTitle = getEmptyViewClickListener().getEmptyViewTitle();
+//        emptyViewTitleTextPaint.getTextBounds(emptyViewTitle, 0, emptyViewTitle.length(), emptyViewText);
+//
+//        Rect emptyViewSubtitleText = new Rect();
+//        String emptyViewSubtitle = getEmptyViewClickListener().getEmptyViewSubtitle();
+//        emptyViewSubtitleTextPaint.getTextBounds(emptyViewSubtitle, 0, emptyViewSubtitle.length(), emptyViewSubtitleText);
+//        emptyViewSubtitleTextHeight = emptyViewSubtitleText.height();
+//
+//        invalidate();
```

Comment:
Can we get rid of the commented out section unless there is an intentional reason to leave it in?